src/lib/UtapiClient.js

```diff
@@ -555,16 +559,19 @@ export default class UtapiClient {
                     ['zadd',
                         generateStateKey(p, 'storageUtilized'),
                         timestamp, actionCounter]);
-
+                // Abort multipart upload does not affect number of objects
+                if (action === 'abortMultipartUpload') {
+                    return true;
+                }
                 // num of objects counter
-                objectsIndex = i * (cmdsLen / paramsArrLen) + 1;
+                objectsIndex = i * (cmdsLen / paramsArrLen) + 2;
```

Comment:
As discussed, either add a comment or define objectIncrementer as cmds[2] or something like that so we know what the magical +2 refers to.